lib/StatusIndicator/styles.scss

```diff
@@ -2,30 +2,55 @@
    Config
    ========================================================================== */
 
-$status-indicator-size: 1em !default;
 $status-indicator-label-color--text: rgb(111, 121, 129) !default;
 
 /* ==========================================================================
    Styles
    ========================================================================== */
 
 .status-indicator {
-  display: inline-block;
-  vertical-align: middle;
   color: $status-indicator-label-color--text;
+  .due-date {
+    color: #8492a6;
+    font-size: 12px;
+  }
+}
+
+.status-indicator--flex {
+  display: flex;
+  align-items: center;
 }
 
 .status-indicator__circle {
-  display: inline-block;
-  vertical-align: middle;
   border-radius: 50%;
-  width: $status-indicator-size;
-  height: $status-indicator-size;
+  width: 12px;
```

Comment:
Is there a reason why we removed the `$status-indicator-size` variable? That can still be used for this 👍